testcontainers-dapr/src/main/java/io/dapr/testcontainers/DaprContainer.java

```diff
@@ -357,52 +399,9 @@ protected void configure() {
 
     withCommand(cmdArray);
 
-    if (configuration != null) {
-      String configurationYaml = CONFIGURATION_CONVERTER.convert(configuration);
-
-      LOGGER.info("> Configuration YAML: \n");
-      LOGGER.info("\t\n" + configurationYaml + "\n");
-
-      withCopyToContainer(Transferable.of(configurationYaml), "/dapr-resources/" + configuration.getName() + ".yaml");
-    }
-
-    if (components.isEmpty()) {
-      components.add(new Component("kvstore", "state.in-memory", "v1", Collections.emptyMap()));
-      components.add(new Component("pubsub", "pubsub.in-memory", "v1", Collections.emptyMap()));
-    }
 
-    if (subscriptions.isEmpty() && !components.isEmpty()) {
-      subscriptions.add(new Subscription("local", "pubsub", "topic", "/events"));
-    }
-
-    for (Component component : components) {
-      String componentYaml = COMPONENT_CONVERTER.convert(component);
-
-      LOGGER.info("> Component YAML: \n");
-      LOGGER.info("\t\n" + componentYaml + "\n");
-
-      withCopyToContainer(Transferable.of(componentYaml), "/dapr-resources/" + component.getName() + ".yaml");
-    }
 
-    for (Subscription subscription : subscriptions) {
-      String subscriptionYaml = SUBSCRIPTION_CONVERTER.convert(subscription);
-
-      LOGGER.info("> Subscription YAML: \n");
-      LOGGER.info("\t\n" + subscriptionYaml + "\n");
-
-      withCopyToContainer(Transferable.of(subscriptionYaml), "/dapr-resources/" + subscription.getName() + ".yaml");
-    }
-
-    for (HttpEndpoint endpoint : httpEndpoints) {
-      String endpointYaml = HTTPENDPOINT_CONVERTER.convert(endpoint);
-
-      LOGGER.info("> HTTPEndpoint YAML: \n");
-      LOGGER.info("\t\n" + endpointYaml + "\n");
-
-      withCopyToContainer(Transferable.of(endpointYaml), "/dapr-resources/" + endpoint.getName() + ".yaml");
-    }
-
-    dependsOn(placementContainer, schedulerContainer);
+    super.start();
```

Comment:
@cicoyle this is definitely not good, be careful with this. If you add a `super.start()` here testcontainers will not be able to manage the lifecycle of the container. Why was this added? I don't agree with this change.